tests/console/zypper_lifecycle.pm

```diff
@@ -184,6 +185,8 @@ fi
 EOF
     script_output $restore;
     select_user_serial_terminal;
+    # Some (older) versions of bash don't take changes to the terminal during runtime into account. Re-exec it.
+    enter_cmd('export PAGER=cat TERM=dumb; stty cols 2048; exec $SHELL') if (is_sle('>=16') && is_s390x);
```

Comment:
This is not the correct place, it should be moved to console login handler

Reply:
svirt does not run the login https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/susedistribution.pm#L907 -> https://openqa.suse.de/tests/17871068#step/zypper_lifecycle/116
But yes this should be part of some generic console setting
This is executed on s390x svirt https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/susedistribution.pm#L951
Maybe move the export into `set_serial_prompt` ?

Reply:
Yup, I would put it there https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/serial_terminal.pm#L139